pkg/apis/enterprisesearch/v1beta1/enterprisesearch_types.go

```diff
@@ -26,11 +26,10 @@ type EnterpriseSearchSpec struct {
 	// Config holds the Enterprise Search configuration.
 	Config *commonv1.Config `json:"config,omitempty"`
 
-	// ConfigRef contains references to Kubernetes Secrets holding the Enterprise Search configuration.
-	// Configuration settings are merged and have prcedence over plain text settings specified in  `config`.
-	// Multiple secrets can be referenced: if duplicate settings exist in multiple secrets,
-	// the last one takes precedence.
-	ConfigRef []ConfigSource `json:"configRef,omitempty"`
+	// ConfigRef contains references to an existing Kubernetes Secret holding the Enterprise Search configuration.
+	// Configuration settings are merged and have precedence over plain text settings specified in `config`.
```

Comment:
```suggestion
	// Configuration settings are merged with and have precedence over settings specified in `config`.
```
Without "with" it seems that they are many of them that are being merged together. 
I think I'd remove "plain text" since the one in configRef are fairly plain texty to me :) Even the example we have is plain text.